topic04-git-introduction/book-git-tutorial/03.Rebasing-Without-Conflicts.md

```diff
@@ -0,0 +1,40 @@
+# Git rebasing
+
+Push the changes by typing `git push <remote> <branch>`, If the branch isn’t out of date, then the changes will push straight up to your github repo, If it is out of date however, you will have to rebase.
+
+## What is rebasing?
+If someone commits a change and your branch is behind, "git rebase" will fast forward your branch to include your changes 
+
+### If you do have to rebase, you will be presented with 3 options :
+```sh
+git rebase --continue
+git rebase --skip
+git rebase --abort
+```
+
+`git rebase --continue` : used when there are no conflicts or the conflicts are solved.  
+
+`git rebase --skip` : skips the file and won't be involved in the rebase.  
+
+`git rebase --abort` : kills the process completely, allowing you to start again.  
+
+## The process of rebasing
+Before you start rebasing, you want to make sure that your local repo is up to date with the one in the cloud. To do this you go to the github page of the repo you forked, and it will say weather your branch is behind or up to date
```

Comment:
I would say something along the lines of `... you want to make sure that your local branch is up to date with the remote master branch. In order to do this, you can pull any updates from the remote to your local master branch by doing a git pull. If there was no updates, that denotes that your branch is up to date...` Follow up with the git checkout master and git pull origin master commands.